dot/parachain/dispute/overseer/message.go

```diff
@@ -81,3 +81,23 @@ type ValidationResult struct {
 	ValidResult   *ValidValidationResult
 	InvalidResult *InvalidValidationResult
 }
+
+type FinalizedBlockNumberResponse struct {
+	Number uint32
+	Err    error
+}
+
+type FinalizedBlockNumberRequest struct {
+	ResponseChannel chan FinalizedBlockNumberResponse
+}
+
+type AncestorsResponse struct {
+	Ancestors []common.Hash
+	Error     error
+}
+
+type AncestorsRequest struct {
+	Hash            common.Hash
+	K               uint32
```

Comment:
What `K` means ?

Reply:
The number of ancestors to request. 

I had worked on the `overseer` messages even before `feat/parachain` had the support for it so I did not document them. I'll be doing that in the final PR for disputes-coordinator.